windows-latest.yml

```diff
@@ -0,0 +1,33 @@
+##############################################################
+# WARNING! This environment file was generated from the
+# conda/meta.yaml file using the metatoenv.py tool.
+# Do not update this file in-place.
+# Use metatoenv.py to create a new file.
+##############################################################
```

Comment:
Are you going to remove this file again once we use the same setup on all OS's?

Reply:
We will not. Rather, we would add one file per platform, since, e.g., Linux requires the docs build dependencies. Or we keep one file for all platforms and do the docs build in a separate job/step (just like in the most recent iteration of the package builds).

Reply:
So is it meant specifically for CI? How does it work with `metatoenv`? Sounds like it needs manual edits. So what changed compared to when we wanted to avoid platform specific files?

Reply:
> How does it work with metatoenv
I used that and then stripped unneeded files.

> Sounds like it needs manual edits
Yes, could potentially be automated.

> So what changed compared to when we wanted to avoid platform specific files?
More specific files? Remember my hesitation about the metatoenv script approach. But obviously neither approach comes without problems. Another (now viable) alternative would be to use `pip` for CI? This should be possible now?